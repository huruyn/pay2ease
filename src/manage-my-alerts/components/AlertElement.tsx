import { Form } from 'react-bootstrap';

export default function AlertElement({ text }: { text: string }) {
  return (
    <div style={{ margin: '20px 5px' }}>
      <div className="d-flex">
        <Form.Check type="checkbox" label={text} />
        <i />
      </div>
      <hr
        style={{
          height: '2px',
          color: 'C4C4C4',
        }}
      />
    </div>
  );
}
