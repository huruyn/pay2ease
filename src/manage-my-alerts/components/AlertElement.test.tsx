import React from 'react';
import AlertElement from './AlertElement';

const ReactTestRenderer = require('react-test-renderer');

describe('AlertElement Component', () => {
  it('matches the snapshot', () => {
    const tree = ReactTestRenderer.create(
      <AlertElement text="test" />,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
