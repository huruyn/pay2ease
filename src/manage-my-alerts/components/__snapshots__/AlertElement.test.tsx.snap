// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AlertElement Component matches the snapshot 1`] = `
<div
  style={
    Object {
      "margin": "20px 5px",
    }
  }
>
  <div
    className="d-flex"
  >
    <div
      className="form-check"
    >
      <input
        className="form-check-input"
        disabled={false}
        type="checkbox"
      />
      <label
        className="form-check-label"
        title=""
      >
        test
      </label>
    </div>
    <i />
  </div>
  <hr
    style={
      Object {
        "color": "C4C4C4",
        "height": "2px",
      }
    }
  />
</div>
`;
